scripts/warn-outside-container

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Was there anything in the script requiring bash now, or would sh still work?

Reply:
Quoting myself

> Also, use bash instead of sh because in POSIX shell echo does not take any options.

Reply:
The alternative is to change all `echo` calls to `printf` (and add `\n` everywhere).